pkg/machine/qemu/machine.go

```diff
@@ -599,27 +599,47 @@ func (v *MachineVM) Start(name string, opts machine.StartOptions) error {
 			_ = v.writeConfig()
 		}
 	}
-	if len(v.Mounts) > 0 {
-		connected := false
-		backoff = 500 * time.Millisecond
-		for i := 0; i < maxBackoffs; i++ {
-			if i > 0 {
-				time.Sleep(backoff)
-				backoff *= 2
-			}
-			state, err := v.State(true)
-			if err != nil {
-				return err
-			}
-			if state == machine.Running && v.isListening() {
-				connected = true
-				break
+	if len(v.Mounts) == 0 {
+		v.waitAPIAndPrintInfo(forwardState, forwardSock, opts.NoInfo)
+		return nil
+	}
+
+	connected := false
+	backoff = 500 * time.Millisecond
+	var sshError error
+	for i := 0; i < maxBackoffs; i++ {
+		if i > 0 {
+			time.Sleep(backoff)
+			backoff *= 2
```

Comment:
Unless I'm doing bad math, which is entirely possible.  If you go through the full cycle, the last sleep will be 16K milliseconds or about 16 seconds.  Do you really want to wait that long?  I'd suggest capping at 5K milliseconds, or maybe 10K

Reply:
This logic predates this change but I see your argument.  I'll reduce `maxBackoffs`

Reply:
It turns out that changing this magic value causes issues.  Let's defer that to another day.